api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -262,7 +262,8 @@ message Passthrough {}
 message MergeExtractorsToBody {}
 
 message HeaderBodyTransform {
-  // When transforming a request, setting this to true will additionally add "queryString", 
-  // "httpMethod" and "path" to the body
+  // When transforming a request, setting this to true will additionally add
+  // "queryString", "queryStringParameters", "multiValueQueryStringParameters",
+  // "multiValueHeaders", "httpMethod" and "path" to the body
```

Comment:
can we clarify that multi value is omitted if there's only one value (to match AWS behavior)?